app/src/androidTest/java/ch/epfl/sdp/blindly/fake_module/FakeUserCacheModule.kt

```diff
@@ -26,7 +26,8 @@ open class FakeUserCacheModule {
         private const val showMe = "Everyone"
         private val passions = listOf("Coffee", "Tea")
         private const val radius = 150
-        private val matches: List<String> = listOf()
+        private val matches: List<String> = listOf("a1", "b2")
+        private val likes: List<String> = listOf("c3", "d4")

```

Comment:
maybe be more explicit about the content of the list rather than just  a1, b2 etc